### DISCUSSION TITLE
full bom available?

### DISCUSSION THREAD

**Discussion by ogg1e** [mebs/CM4-NAS#13] [General]
Is there a full bom available for all the parts? Or at least links to the power connectors needed?

**Comment by mebs** [comment]
I have listed the most specific components in the Readme. The rest are all generic but you would have to open the project to look at resistor values and such.
Connectors are generic JST connectors or pin headers, but you can put whatever you want for as long as the pitch is the standard 2.54mm.

**Reply by ogg1e** [reply]
ok. great. What about the oled and buttons? I don't see any details there. I think I might make a BOM that can then be imported to digikey, mouser etc.

**Reply by mebs** [reply]
The OLED screen and buttons are not on the board, they're peripherals you can replace with whatever you want so they don't need to be on the BOM. 
I used a SSD1306 OELD screen but any I2C screen will work.

**Reply by ogg1e** [reply]
The reason I asked was so I could source ones that would fit in the 3d printed case.

**Reply by mebs** [reply]
I designed the lid of the case to work with a 128x32 SSD1306 OLED screen like this one:
https://www.aliexpress.com/item/32798439084.html
I think Adafruit used to have them also but I couldn't find it anymore, the only 128x32 screens they have have a different form factor.
